docs/how-tos/providing-images-for-img.rst

```diff
@@ -36,64 +25,40 @@ First, we need a public table for storing the files.
 Let's assume this table contains an image of two cute kittens with id 42.
 We can retrieve this image in binary format from our PostgREST API by requesting :code:`/files?select=blob&id=eq.42` with the :code:`Accept: application/octet-stream` header.
 Unfortunately, putting the URL into the :code:`src` of an :code:`<img>` tag will not work.
-That's because browsers do not send the required header.
-
-Luckily, we can configure our :doc:`Nginx reverse proxy <../admin>` to fix this problem for us.
-We assume that PostgREST is running on port 3000.
-We provide a new location :code:`/files/` that redirects requests to our endpoint with the :code:`Accept` header set to :code:`application/octet-stream`.
-
-.. code-block:: nginx
-
-   server {
-     # rest of reverse proxy and web server configuration
-     ...
-
-     location /files/ {
-       # /files/<id>/* ---> /files?select=blob&id=eq.<id>
-       rewrite /files/([^/]+).*  /files?select=blob&id=eq.$1  break;
-       # if id is missing
-       return 404;
-       # request binary output
-       proxy_set_header Accept application/octet-stream;
-       # usual proxy setup
-       proxy_hide_header Content-Location;
-       add_header Content-Location /api/$upstream_http_content_location;
-       proxy_set_header  Connection "";
-       proxy_http_version 1.1;
-       proxy_pass http://localhost:3000/;
-     }
```

Comment:
It looks like the examples do not need the reverse proxy anymore. Maybe add another section below, mentioning that `raw-media-types` won't be needed if the :ref:`Accept: application/octet-stream` header is added in Nginx?

Reply:
Oh, so we don't need Nginx at all now right? Looks much better this way.

> that raw-media-types won't be needed if the Accept: application/octet-stream header is added in Nginx?

I think, it's fine to leave Nginx out for this use case.